setup.py

```diff
@@ -1,87 +1,76 @@
 #!/usr/bin/env python
-descr = """\
```

Comment:
Sorry... I know it looks like I defaced your `setup.py`, but I tried to make it a little simpler and removed the bogus dependency on `numpy` as install time.